tensornetwork/ncon_interface.py

```diff
@@ -72,10 +76,30 @@ def ncon(tensors: Sequence[Tensor],
         `tensornetwork.config.default_backend`.
 
     Returns:
-      A `Tensor` resulting from the contraction of the tensor network.
+      The result of the contraction. The result if returned in a `Node` 
+        if all elements of `tensors` are `BaseNode` objects, else
+        it is returned as a `Tensor` object.
```

Comment:
Your typing hint suggests that it will always be a `Tensor`.

Reply:
fixed it